src/styles/_catalog.scss

```
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
```

```suggestion
  grid-template-columns: 200px;
```
THis rule could be shorter)